metron-platform/metron-indexing/README.md

```diff
@@ -15,6 +15,12 @@ Indices are written in batch and the batch size and batch timeout are specified
 [Sensor Indexing Configuration](#sensor-indexing-configuration) via the `batchSize` and `batchTimeout` parameters.
 These configs are variable by sensor type.
 
```

Comment:
Do we care, here in particular, that we have different required fields listed?  Should this not be cumulative and include the required fields coming out of the parsers?

Reply:
So, strictly speaking messages really only require `source.type` (which I typo'd) and `timestamp` (which I should add).  I'll fix that, but did I miss anything?